src/action/wallet.js

```diff
@@ -53,7 +53,7 @@ class WalletAction {
    * @param {number} options.index The seed index
    */
   setRestoreSeed({ word, index }) {
-    this._store.seedMnemonic[index] = word;
+    this._store.seedMnemonic[index] = word.trim();
```

Comment:
This is a really simple fix. A unit tests to reproduce/fix error states would be useful though.

Reply:
Added! Just realized I misnamed the branch as well.. next time will name `dev/trim-seed-words`.

Reply:
Awesome!